### PR TITLE
Python and html fixes (rebased)

### PR DESCRIPTION
**Pull request by bertm**
This is a rebase of #17, including the fixups added there, with one extra trivial commit at the end.

### CODE REVIEW

File: pages/help.py [hunk 1]
```diff
@@ -489,13 +488,12 @@ def get_content(self):
...
-https://wiki.freenetproject.org/Installing/Windows#.27Download
-.2Fupload_queue_database_corrupted.21.27_.28When_using_Kaspersky_on_Windows_7
-.29). We recommend you turn off Kaspersky during install and during node
-startup, and exclude the directory you installed Freenet in (most likely
-C:\Program Files\Freenet or C:\Program Files (x86)\Freenet).
-""")),
+Kaspersky can be a problem with Freenet. See [here][url_kaspersky].
+We recommend you turn off Kaspersky during install and during node startup, and exclude the directory you installed Freenet in (most likely {freenet_install_dir_1} or {freenet_install_dir_2}).
+""").format(freenet_install_dir_1="C:\Program Files\Freenet",
+            freenet_install_dir_2="C:\Program Files (x86)\Freenet") + "\n\n" + """
```
Comment (Thynix) [inline pages/help.py:494]: Does Windows really not localize this? Also this is dated from when Freenet was a system service; it [defaults to](https://github.com/freenet/wininstaller-innosetup/blob/master/FreenetInstall_InnoSetup.iss#L21) `LocalAppData%\Freenet` now. That's for fixing separately though.

**Comment by Thynix** [comment]
Deployed, thanks!

**Comment by Thynix** [comment]
This reduces the vertical space before the donation bar too, which is great.

File: pages/help.py (hunk 1 above)
Comment (bertm) [inline pages/help.py:494]: The firectories are indeed localized, however [for most languages](https://en.wikipedia.org/wiki/Program_Files) it's plain "Program Files". Given the text mentions "most likely", there's no point in translating this. Then again users will probably know what "Program Files" means, but may not understand the reverse translation if they happen to be using an international Windows installation.

**Pull request merged**